app/helpers/application_helper.rb

```diff
@@ -21,6 +21,10 @@ def bootstrap_class_for(flash_type)
     bootstrap_classes.fetch(flash_type, custom_flash(flash_type))
   end
 
+  def active_class(path)
```

Comment:
Can we include a unit test for this please?

Reply:
Sure :)